canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -514,6 +517,19 @@ class FlexibleTable extends React.Component {
 			)
 			: null;
 
+		let tableHeight = 0;
+		const multiSelectEditRowsRem = 2 * REM_HEADER_HEIGHT; // multi-select adds two rows when selectedEditRow
+		const multiSelectEditRowsPixels = multiSelectEditRowsRem * ONE_REM_HEIGHT;
+		if (this.state.rows !== -1 && this.state.tableHeight) {
+			const remHeight = parseInt(this.state.tableHeight.match(/^[0-9]{1,2}/i)[0], 10);
+			tableHeight = (remHeight - (this.props.selectedEditRow ? multiSelectEditRowsRem : 0)) * ONE_REM_HEIGHT;
+		} else if (this.state.rows === -1) {
+			if (this.state.dynamicHeight && this.state.dynamicHeight !== -1) {
+				tableHeight = this.state.dynamicHeight - (this.props.selectedEditRow ? multiSelectEditRowsPixels : 0);
+			} // else how do we handle this.state.tableHeight = "100vh"?
+		}
+		console.log(tableHeight);
```

Comment:
Please remove this console.log statement

Reply:
😂 ... happens when you push code in middle of the night